docs/app/Examples/collections/Menu/Content/SubMenu.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Button, Icon, Input, Menu } from 'stardust'
+
+const {Item} = Menu
+
+const SubMenu = () => {
+  return (
+    <Menu vertical>
+      <Item>
+        <Input placeholder="Search..." />
+      </Item>
+
+      <Item>
+        Home
+
+        <Menu>
```

Comment:
Take a look there. `<Menu>` produces `ui menu`, but for correct look it might be only `menu`.

I'm creating `Menu.Menu`, `Menu.Sub`? :hankey: